docs/components/Banner.js

```diff
@@ -0,0 +1,122 @@
+import * as React from 'react';
+import { css } from 'react-emotion';
+
+import * as Constants from '~/common/constants';
+
+const STYLES_ALERT = css`
+  padding: 16px;
+  border-radius: 4px;
+  margin-bottom: 24px;
+  line-height: 1.4;
+  color: ${Constants.colors.white};
+  background: ${Constants.colors.black};
+  box-shadow: 0 1px 4px rgba(0, 0, 0, 0.2);
+  overflow: hidden;
+`;
+
+const STYLES_ALERT_BOLD = css`
+  color: ${Constants.colors.white};
+  font-family: ${Constants.fontFamilies.demi};
+`;
+
+const STYLES_CLOSE_BUTTON = css`
+  float: right;
+  position: relative;
+  width: 20px;
+  height: 20px;
+  background: ${Constants.colors.black};
+  border: 0;
+  ::before,
+  ::after {
+    position: absolute;
+    left: 0px;
+    width: 18px;
+    height: 2px;
+    content: '';
+    background-color: ${Constants.colors.white};
+  }
+  ::before {
+    transform: rotate(45deg);
+  }
+  ::after {
+    transform: rotate(-45deg);
+  }
+  :focus {
+    outline: none;
+  }
+  :hover {
+    cursor: pointer;
+  }
+`;
+
+export default class Banner extends React.Component {
+  constructor(props) {
+    super(props);
+    let isHidden;
+    if (typeof window !== 'undefined') {
+      isHidden = localStorage.getItem('isBannerHidden');
+    } else {
+      isHidden = false;
+    }
+    this.state = {
+      isHidden,
+    };
+  }
```

Comment:
we should change this to default to `isHidden: false` because when using ssr (or static export) we will always set `isHidden` to `false`, leading to an initial flash of the banner being visible before disappearing. most likely people will hide it the first or second time they see it, so it's best to assume it's hidden.

i think with banners like this it's usually better to position them in a way where they don't impact layout because we don't know on server if they are visible or not. but that's more work right now and not absolutely necessary, so i'll make this one change locally and shipit. thanks!